Source/MoreDotNet.Test/Extensions/Common/RandomExtensions/OneOfTests.cs

```diff
@@ -0,0 +1,40 @@
+﻿namespace MoreDotNet.Tests.Extensions.Common.RandomExtensions
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Linq;
+
+    using MoreDotNet.Extensions.Common;
+
+    using Xunit;
+
+    public class OneOfTests
+    {
+        private int[] numbersArr = new[] { 1, 2, 3, 4, 5, 6 };
```

Comment:
- Do not use shorthands.
- Use meaningful names.
- you can do: `private int[] numbersArr = { 1, 2, 3, 4, 5, 6 };`